src/main/java/org/fenixedu/academic/service/services/bolonhaManager/EditCourseGroup.java

```diff
@@ -27,13 +27,22 @@
 import org.fenixedu.academic.service.services.exceptions.FenixServiceException;
 
 import pt.ist.fenixframework.Atomic;
+import pt.ist.fenixframework.DomainObject;
 import pt.ist.fenixframework.FenixFramework;
 
 public class EditCourseGroup {
 
-    @Atomic
```

Comment:
This way wouldn't the old method no longer be @Atomic? That can cause issues with other methods invoking it.